CMakeLists.txt

```diff
@@ -21,7 +21,11 @@ if (NOT DEFINED CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "")
 endif ()
 
-set(HUNTER_CONFIGURATION_TYPES ${CMAKE_BUILD_TYPE} CACHE STRING "")
+if(${CMAKE_BUILD_TYPE} MATCHES "^RelWithDebInfo$")
+  set(HUNTER_CONFIGURATION_TYPES "Release" CACHE STRING "")
+else()
+  set(HUNTER_CONFIGURATION_TYPES "${CMAKE_BUILD_TYPE}" CACHE STRING "")
```

Comment:
If we're not fixing the problem in the protobuf repo itself, I think it makes sense to just prohibit build types other than Release, Debug and RelWithDebInfo, since protobuf is known to break with non-standard build types.

Reply:
`set(HUNTER_CONFIGURATION_TYPES "${CMAKE_BUILD_TYPE}")` was intended to build only one of two hunter default types `Debug;Release`, this check fixes `RelWithDebInfo`.